ph5/core/ph5api.py

```diff
@@ -800,101 +814,79 @@ def query_das_t(self,
-                            'raw_file_name_s': row['raw_file_name_s'],
-                            'receiver_table_n_i': row['receiver_table_n_i'],
-                            'response_table_n_i': row['response_table_n_i'],
-                            'sample_count_i': row['sample_count_i'],
-                            'sample_rate_i': row['sample_rate_i'],
-                            'sample_rate_multiplier_i':
-                                row['sample_rate_multiplier_i'],
-                            'stream_number_i': row['stream_number_i'],
-                            'time/ascii_s': row['time/ascii_s'],
-                            'time/epoch_l': row['time/epoch_l'],
-                            'time/micro_seconds_i':
-                                row['time/micro_seconds_i'],
-                            'time/type_s': row['time/type_s'],
-                            'time_table_n_i': row['time_table_n_i']
-                            }
-                das.append(row_dict)
+            )
         elif check_samplerate is False:
-            for row in tbl.where(
-                 '(channel_number_i == '
-                 + str(chan) + ' )&(epoch_i+micro_seconds_i/1000000 >= '
-                 + str(start_epoch) +
-                 '-sample_count_i/sample_rate_i/sample_rate_multiplier_i)'
-                 '&(epoch_i+micro_seconds_i/1000000 <= '
-                 + str(stop_epoch) + ')'
-                 ):
-                row_dict = {'array_name_SOH_a': row['array_name_SOH_a'],
-                            'array_name_data_a': row['array_name_data_a'],
-                            'array_name_event_a': row['array_name_event_a'],
-                            'array_name_log_a': row['array_name_log_a'],
-                            'channel_number_i': row['channel_number_i'],
-                            'event_number_i': row['event_number_i'],
-                            'raw_file_name_s': row['raw_file_name_s'],
-                            'receiver_table_n_i': row['receiver_table_n_i'],
-                            'response_table_n_i': row['response_table_n_i'],
-                            'sample_count_i': row['sample_count_i'],
-                            'sample_rate_i': row['sample_rate_i'],
-                            'sample_rate_multiplier_i':
-                                row['sample_rate_multiplier_i'],
-                            'stream_number_i': row['stream_number_i'],
-                            'time/ascii_s': row['time/ascii_s'],
-                            'time/epoch_l': row['time/epoch_l'],
-                            'time/micro_seconds_i':
-                                row['time/micro_seconds_i'],
-                            'time/type_s': row['time/type_s'],
-                            'time_table_n_i': row['time_table_n_i']
-                            }
-                das.append(row_dict)
+            if col_srm_exist is True:
+                numexprstr = (
```

Comment:
What is numexprstr? This seems like a lot of additional refactoring that is occurring in PH5API. Is this directly related to this issue?

Reply:
Since 
```
            for row in tbl.where(....):
                row_dict = {....}
```
has been repeated a lot and the code took too many lines. I assign the expression inside `tbl.where()` to `numexprstr` so we don't need to repeat that part. It helps developer to maintain the code easier.

Reply:
This a good refactor and should be left in the code. It makes this function more concise.